testscommon/marshallerMock.go

```diff
@@ -0,0 +1,56 @@
+package testscommon
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+// ErrMockMarshalizer -
+var ErrMockMarshalizer = errors.New("MarshalizerMock generic error")
+
+// ErrNilObjectToMarshal -
+var ErrNilObjectToMarshal = errors.New("nil object to serialize from")
+
+// MarshalizerMock that will be used for testing
+type MarshalizerMock struct {
```

Comment:
Marshalizer -> Marshaller on all occurrences on this file?

Reply:
done